src/EcommerceStatsBundle/README.md

```diff
@@ -0,0 +1,17 @@
+# eCommerce Statistics Bundle
+
+This CMS bundle serves chameleon's shop statistic stuff.
+
+After installation, ensure to create these sym links:
+
+    $ composer run-script post-install-cmd
```

Comment:
This is basic (Symfony) knowledge.

Are these paths (/web/bundles) used here?

Reply:
True, removed that part. I left the part about `snippets-cms` in there though.